webpack/webpack.extension.config.js

```diff
@@ -38,14 +38,13 @@ const inOutConfig = isExtension
 
 module.exports = {
   ...inOutConfig,
-  devtool: "source-map",
+  devtool: "cheap-module-source-map",
```

Comment:
Any particular reason for this change?

The [docs advise](https://webpack.js.org/configuration/devtool/) sticking with `source-map` for production builds 🤔

Reply:
Someone mentioned in stack-overflow that source-map are using `eval` and the solution was `cheap-module-source-map`. This was when I worked on CSP issue. This is not the solution of the issue, so I will revert it. Thanks